Felienne/hedy
# Users are always able to visit the for-teachers page

Provide a general summary of your changes in the Title of the PR

**Description**

Currently, users are always able to visit the for-teachers page, even when they are not a teacher, by directly navigating to the url. The shown class information is empty and the users are unable to create a class but still: this is undesirable behavior. In this PR we add an additional check in `app.py` to verify if the user is teacher when requesting the for-teachers page. If not, the user is redirected to `index.html`.

**Fix for**

Describe what issue or discussion your pr concerns

**How to test**

Try to visit the for-teachers page directly using this link: [](http://0.0.0.0:5000/for-teachers). If you're currently not logged in as a teacher you should be re-directed to the homepage. Otherwise the for-teachers page should be shown as expected.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
